Client/src/Components/Profile/PermissionsSection/PermissionsTab.tsx

```diff
@@ -0,0 +1,76 @@
+import * as Yup from 'yup'
+
+import { Box, Button, Grid, Typography } from '@material-ui/core'
+import { FastField, Form, Formik } from 'formik'
+import { fetchAllAdmins, updatePermissions } from '../../../Remote/Endpoints/PermissionsEndpoint'
+import { AdminList } from './AdminList'
+
+import { MuiTextFieldFormik } from '../../Forms/FormikFields'
+import React, { useEffect } from 'react'
+import { classStyles } from '../../../appTheme'
+import { useState } from 'react'
+
+export default function PermissionsTab() {
+
+  const validationSchema = Yup.object().shape({
+    email: Yup.string()
+      .email().required("Email Address")
+  })
+
+  const [adminListState, setAdminListState] = useState([])
+
+  const handleAddAdmin = async (newAdmin: any) => {
+    await updatePermissions({
+      email: newAdmin.email,
+      operation: "add"
+    })
+    const adminListState = await fetchAllAdmins()
+    setAdminListState(adminListState)
+  }
+
+  const handleRemoveAdmin = async (newAdmin: string) => {
+    await updatePermissions({
+      email: newAdmin,
+      operation: "remove"
+    })
+    const adminListState = await fetchAllAdmins()
+    setAdminListState(adminListState)
+  }
+
+  useEffect(() => {
+    const callListAdminStates = async () => {
+      const adminListState = await fetchAllAdmins()
+      setAdminListState(adminListState)
+    }
+    callListAdminStates()
+  }, [])
+
+  return (
+    <Grid>
+      <Formik
+        initialValues={{ email: "" }}
+        validationSchema={validationSchema}
+        onSubmit={handleAddAdmin}
+      >
+        <Form>
+          <Box className={classStyles().defaultBorder} style={{ width: "100%" }} >
```

Comment:
![image](https://user-images.githubusercontent.com/22376976/107598294-57d51100-6beb-11eb-8727-cb16bab65bec.png)
the buttons seems off, the email doesnt have to be this wide

Reply:
the button needs to be vertically centered + should be close to the textfield

Reply:
also text should have been regular font, no header